lib/go-tc/deliveryservices.go

```diff
@@ -1726,6 +1736,12 @@ func (ds DeliveryServiceV4) Upgrade() DeliveryServiceV5 {
 		copy(upgraded.MatchList, *ds.MatchList)
 	}
 	copy(upgraded.TLSVersions, ds.TLSVersions)
+	if len(ds.RequiredCapabilities) == 0 {
+		upgraded.RequiredCapabilities = make([]string, 0)
```

Comment:
This line duplicates the effect of line 1703 under the conditions checked on line 1739